src/elit/boot_asset_fingerprint.clj

```diff
@@ -0,0 +1,56 @@
+(ns elit.boot-asset-fingerprint
+  {:boot/export-tasks true}
+  (:require [boot.core :as boot]
+            [clojure.java.io :as io]
+            [elit.dir-writer :as writer]
+            [elit.fingerprint :as fingerprint]))
+
+(def default-extensions [".html"])
+
+(defn asset-fingerprint*
+  [files out-dir {:keys [asset-root asset-host extensions path->file skip?] :as opts}]
+  (let [file-writer (writer/->TmpDirWriter out-dir)]
+    (loop [[{:keys [path] :as file} :as content-files] files
```

Comment:
Very minor syntax suggestion, you may prefer to go with the existing, but just in case....
You could use `& more-files` here, then you don't need the `rest` call below?

Reply:
👍